stix_shifter_modules/splunk/stix_translation/json/from_stix_map.json

```diff
@@ -83,9 +83,10 @@
     "file": {
         "cim_type": "endpoint",
         "fields": {
-            "hashes.MD5": "file_hash",
-            "hashes.'SHA-1'": "file_hash",
-            "hashes.'SHA-256'": "file_hash",
+            "hashes.MD5": ["Hashes.MD5"],
+            "hashes.SHA1": ["Hashes.SHA1"],
+            "hashes.SHA256": ["Hashes.SHA256"],
+             "hashes.IMPHASH": ["Hashes.IMPHASH"],
```

Comment:
IMPHASH needs to be in `windows-pebinary-ext` extension of file object not in hashes.

```
"type": "file",
"extensions": {
    "windows-pebinary-ext": {
```
REF: https://docs.google.com/document/d/167aIyr5BIAJJORzjT11U25cGSBJ0cBNSdkheNJFz6l8/edit#heading=h.gg5zibddf9bs